src/pages/privacy.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+import Layout from '../components/layout';
+import ContentContainer from '../components/content-container';
+
+const PrivacyPolicy = () => {
+  const heading = 'Privacy Policy';
+
+  const content = (
+    <p>
+      Graasp Insights is developed and maintained by the{' '}
```

Comment:
Same as above, but it seems that these longer texts will be complicated to include with `t()` so let's leave it for a different PR.